### ISSUE TITLE
Please add a link to the 2.1 specification from the understanding documents

### ISSUE THREAD

**Issue by lauracarlson** [w3c/wcag#3293]
The previous versions of WCAG had a link to the specification, [for example 2.0's 1.2.1 ](https://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-av-only-alt.html) and  [2.1's 1.2.2](https://web.archive.org/web/20190713065229/https://www.w3.org/WAI/WCAG21/Understanding/audio-only-and-video-only-prerecorded). The 2.1 understanding docs currently lack a link, [for example 2.1's 1.2.1 ](https://www.w3.org/WAI/WCAG21/Understanding/audio-only-and-video-only-prerecorded.html).

Many people will find the understanding doc from search and can't use back button.

**Comment by patrickhlauke** [comment]
Closing this as a duplicate of https://github.com/w3c/wcag/issues/3289
